packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -432,6 +447,12 @@ function* watchAddMessage() {
   yield takeEvery(addMessage, ({ payload }) => fetchChatIfNecessary(payload))
 }
 
+function* watchFetchChat() {
```

Comment:
Could we reuse fetchChatIfNecessary instead of adding a new saga here? Pass a bustCache flag or something?